boa_parser/src/parser/mod.rs

```diff
@@ -119,7 +114,7 @@ pub struct Parser<R> {
 
 impl<R> Parser<R> {
     /// Create a new `Parser` with a reader as the input to parse.
-    #[inline]
+
```

Comment:
Don't like the empty line between the comment and function
I'm gonna stop spamming, there's 4 more below this one